cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala

```diff
@@ -93,11 +93,11 @@ This constraint will create an accompanying index.
 //
 
 CREATE CONSTRAINT uniqueness ON (p:Person)
-       ASSERT p.age IS UNIQUE
+       ASSERT (p.firstname, p.age) IS UNIQUE
 ###
 
-Create a unique property constraint on the label `Person` and property `age` with the name `uniqueness`.
-If any other node with that label is updated or created with a `age` that
+Create a unique property constraint on the label `Person` and properties `firstname` and `age` with the name `uniqueness`.
+If any other node with that label is updated or created with a `firstname` and `age` combination that
 already exists, the write operation will fail.
 This constraint will create an accompanying index.
```

Comment:
```suggestion
This constraint creates an accompanying index.
```
This should really be updated everywhere, but I understand if that is out of scope for the PR.

Reply:
I'll update those you have marked at least (and then see if there are any left in this file)